### ISSUE TITLE
Compatibility with 2.9x

### ISSUE THREAD

**Issue by tnakagome** [nolanyee/blender-path-extrude#2]
Hello.

Attached blend file has a circle and a spiral. If you use Blender 2.8x and extrude the circle along the spiral, it will create a good-looking spring-like object. However, If I use Blender 2.9x and perform the same operation, the resulting object appears pinched at multiple positions.

[ExtrusionTest.blend.zip](https://github.com/nolanyee/blender-path-extrude/files/5260310/ExtrusionTest.blend.zip)

I am not sure if the problem is in Blender itself, but it would be great if this tool could be made compatible with Blender 2.9x.

Thanks.

**Comment by nolanyee** [comment]
The rotate method in Blender 2.9 is reversed for some reason. I updated the code to work for Blender 2.8 and 2.9.

**Comment by tnakagome** [comment]
Perfect. Thank you!

**Comment by tnakagome** [comment]
FYI, the issue was fixed in Blender 2.91. So I modified my code as follows:

```
            if bpy.app.version[0] == 2 and bpy.app.version[1] == 90:
```

**Comment by nolanyee** [comment]
Thanks for letting me know. I updated my code accordingly.
